### ISSUE TITLE
Verify component is compatible with screen readers

### ISSUE THREAD

**Issue by alexberazouski** [vaadin/vaadin-app-layout#21]
* Announce page content after navigation

**Comment by alexberazouski** [comment]
Screen readers do not announce if the page content was changed (neither with iron-pages, nor with with Vaadin Router)
JAWS doesn't read anything from the page on page reload in IE11

**Comment by DiegoCardoso** [comment]
Currently, it doesn't announce when a navigation occurs. From my small research, the main reason is that we are not letting the screen readers know of changes in other areas of the page when a tab is selected. In ARIA, there's a property (`aria-live`) on which you can tell to the screen reader that some area in the page is likely to change. 
In my tests, I added `aria-live="polite"` to the content wrapper, which waits for the screen reader to be idle in order to announce the new content.

My findings:
| Screen reader | Works | Doesn't work |
|---------------|--------|--------------|
| JAWS | Chrome, Firefox | Edge, IE11 |
| NVDA | Chrome, Firefox | Edge,IE11 |
| Voice Over | Safari | Chrome, Firefox |
